subsys/power/power.c

```diff
@@ -89,12 +81,34 @@ void pm_power_state_force(enum power_states state)
 #endif
 }
 
+static inline void pm_state_notify(bool entry)
```

Comment:
I guess 'entry' here is for the action of entering (vs. exiting), a bit confusing

Reply:
yep, agree that is a little bit confusing. Any suggestion ?

Reply:
maybe entering_state or just wrap this with two functions `pm_state_enter_notify()` and `pm_state_exit_notify()`

Reply:
this is an internal function, I think I'll just change the parameter's name. It's possible that I need to control access to `pm_state` so I want to avoid redundant code.

Reply:
The name is fine, but it could be documented to eliminate questions.